src/utils/table-helpers.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Calculates list of options for 'Rows per page' selection in a
+ * TablePagination component to account for a max number of rows
+ * @param maxRows number Cap on number of rows to display at once
+ * @param totalRows number Total number of rows in table
+ * @returns array List of options
+ */
+export const getRowPageOptions = (
```

Comment:
This is a great idea, but it needs a little bit of tweaking to work correctly with the existing tables (from the Requests page):

<img width="880" alt="image" src="https://user-images.githubusercontent.com/73488256/184120551-104337d8-4ba4-4437-9236-b52fccd54478.png">

And now that this is in place, it reveals an issue I hadn't noticed before. If you select a command on the blocklist modal and you're not on the first page, it immediately flips back to the first page. I imagine this could be annoying for someone trying to block a number of commands.